disperser/dataapi/server.go

```diff
@@ -69,6 +69,12 @@ type (
 		Data []*BlobMetadataResponse `json:"data"`
 	}
 
+	UnsignedBatches struct {
```

Comment:
Can we rename `UnsignedBatches` to `OperatorNonsigningPercentage`? (and fix other places accordingly)